src/Infrastructure/Array/include/ESMCI_Array.h

```diff
@@ -385,6 +385,8 @@ namespace ESMCI {
     const int *getTotalLBound()             const {return totalLBound;}
     const int *getTotalUBound()             const {return totalUBound;}
     int getTensorCount()                    const {return tensorCount;}
+    int getReplicatedDimCount()             const
```

Comment:
Note that I don't have a return code (rc) for this function, since nothing can currently lead to it failing. I was thinking of this as just a small step beyond a simple getter. Does that seem reasonable, or should this have a place-holder rc?